io/src/main/scala/fs2/io/file/file.scala

```diff
@@ -119,4 +122,81 @@ package object file {
     Stream
       .resource(Watcher.default(blocker))
       .flatMap(w => Stream.eval_(w.watch(path, types, modifiers)) ++ w.events(pollTimeout))
+
+  /**
+    * Creates a stream of [[Path]]s inside a directory.
+    */
+  def streamDirectory[F[_]: Sync: ContextShift](blocker: Blocker, path: Path): Stream[F, Path] =
```

Comment:
`s/streamDirectory/directoryStream/`? Or `directoryContents`? I'm not keen on `streamDirectory` as it reads like a command to me.

Reply:
I went with `directoryStream` as this is closer to the underlying Java naming.